ui/app/services/user-settings.js

```diff
@@ -0,0 +1,6 @@
+import Service from '@ember/service';
+import localStorageProperty from 'nomad-ui/utils/properties/local-storage';
+
+export default Service.extend({
+  pageSize: localStorageProperty('nomadPageSize', 25),
+});
```

Comment:
I appreciate this facility for centralising user settings and how it provides a nice abstraction for the possible wonderful future where user data can magically persist outside a particular client 🤩🤞